tools/configure_server_settings.rb

```diff
@@ -64,43 +65,45 @@ def types_valid?(old_val, new_val)
   end
 end
 
-server = MiqServer.where(:id => opts[:serverid]).take
-unless server
+server_list = opts[:all_servers] ? MiqServer.all : MiqServer.where(:id => opts[:serverid]).take
```

Comment:
I believe `.take` will return one server here I don't think you can call `.each` below.  Maybe just drop the `.take`.